spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -307,6 +307,7 @@ def main(argv: Sequence[str]):
     # check if warp_curve2straight and warp_straight2curve already exist (i.e. no need to do it another time)
     cache_sig = cache_signature(
         input_files=[fname_in, fname_seg],
+        input_params={"version": __version__},
     )
```

Comment:
Another (probably out of scope) question: Why is caching handled by the callers to `sct_straighten_spinalcord`, but not `sct_straighten_spinalcord` itself? 

Currently, if `sct_straighten_spinalcord` is run standalone, no cache files will be generated, which I found a bit odd? Say if `sct_straighten_spinalcord` generated the cache (and did nothing if the cache files matched the inputs), then that might provide a unified interface for generating + checking the cache files... :thinking:

Reply:
Very good point! Yes, it would be a better design for `sct_straighten_spinalcord` to handle the caching itself, instead of each caller. I think that's out of scope for this quick-fix PR, but I can open a follow-up issue about it.

(Although: I think re-using a tweaked result would be a bad idea: that would make things very hard to reproduce. Ideally the caching mechanism would exactly capture all the inputs that could influence the result and make sure they're unchanged.)